tests/Checker/Check_Context_Tests.php

```diff
@@ -12,11 +12,11 @@ public function set_up() {
 		parent::set_up();
 
 		$this->plugin_name   = basename( TESTS_PLUGIN_DIR );
-		$this->check_context = new Check_Context( WP_PLUGIN_DIR . '/' . $this->plugin_name . '/plugin-check.php' );
+		$this->check_context = new Check_Context( WP_PLUGIN_DIR . '/' . plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE ) );
```

Comment:
This code is a bit unnecessarily complicated now. It now relies on `plugin_basename()`, when really this is just about the main file.
```suggestion
		$this->check_context = new Check_Context( WP_PLUGIN_CHECK_MAIN_FILE );
```